support-frontend/assets/helpers/productPrice/paperSavingsVsRetail.ts

```diff
@@ -0,0 +1,37 @@
+import type { FulfilmentOptions } from 'helpers/productPrice/fulfilmentOptions';
+import {
+	Collection,
+	HomeDelivery,
+} from 'helpers/productPrice/fulfilmentOptions';
+import { ActivePaperProductTypes } from 'helpers/productPrice/productOptions';
+import type {
+	CountryGroupPrices,
+	ProductPrices,
+} from 'helpers/productPrice/productPrices';
+
+function getSavingsForFulfilmentOption(
+	prices: CountryGroupPrices,
+	fulfilmentOption: FulfilmentOptions,
+) {
+	return ActivePaperProductTypes.map((productOption) => {
+		const price = prices[fulfilmentOption]?.[productOption]?.Monthly?.GBP;
+		return price?.savingVsRetail ?? 0;
+	});
+}
+
+function getMaxSavingVsRetail(
```

Comment:
We previously had a lot of duplicated functions for getting product prices for paper, that did the same thing (or even directly called) functions used on other checkouts. With the simplifying of product price lookup via selectors I have removed those duplicated functions and moved these remaining ones, which are specific to displaying paper price info, into a file with a more specific name.